functions/Test-DbaLastBackup.ps1

```diff
@@ -259,7 +268,37 @@ Restores data and log files to alternative locations and only restores databases
 					}
 					
 					$lastbackup = Get-DbaBackupHistory -SqlServer $sourceserver -Databases $dbname -LastFull -IgnoreCopyOnly:$ignorecopyonly
```

Comment:
I removed that as I thought we had already got the backup history into LastBackup? Maybe I missed why you did -last here? I can add it back in, I changed it to LastBackup since I had updated the path in that one for the copy. I think I need to rework that bit anyway to handle multiple files.